fuse_ll/src/fuse/mount.rs

```diff
@@ -10,21 +11,157 @@ use std::path::Path;
 
 use param::*;
 
+pub struct MountOption {
+    pub name: String,
+    pub parser: fn(&mut FuseMountArgs, &MountOption, &str),
+    pub regex: Regex,
+    #[cfg(target_os = "linux")]
+    pub flag: Option<u64>,
+    #[cfg(target_os = "macos")]
+    pub flag: Option<i32>,
+    #[cfg(target_os = "macos")]
+    pub fuse_flag: Option<u64>,
+}
+
+pub fn get_all_options() -> String {
+    let options: Vec<_> = get_mount_options().iter().map(|x| x.name.clone()).collect();
+    options.join(",")
+}
+
+pub fn options_validator(option: String) -> Result<(), String> {
+    let ops: Vec<_> = option.split(',').collect();
+    let ret = ops
+        .iter()
+        .all(|&op| get_mount_options().iter().any(|x| x.regex.is_match(&op)));
```

Comment:
ops is used nowhere else. Could we merge them into one statement?

Reply:
Same as last one.